gateway/1.5/modules/ROOT/pages/index.adoc

```diff
@@ -7,6 +7,12 @@ endif::[]
 
 Anypoint Flex Gateway is an Envoy-based, ultrafast lightweight API gateway designed to manage and secure APIs running anywhere. Built to seamlessly integrate with DevOps and CI/CD workflows, Flex Gateway delivers the performance required for the most demanding applications and microservices while providing enterprise security and manageability across any environment.
 
+Watch the Flex Gateway Product Spotlight video to see a quick overview of Flex Gateway.
+ 
+++++
+include::_partials/product-spotlight-video.html[]
```

Comment:
Can videos have alt text? If so, you should probably add something so that screen readers can accurately describe the video.

Reply:
I assume you can, but I do not know if it makes sense.

I think it might be redundant to try to capture what is on the screen. Nothing we write will provide more context than the video itself, and then we also have the text below as more overview of the product. 

The video also has an intro text, so I don't think we need to add anything saying something like "video here."